hardware.Dockerfile

```diff
@@ -2,8 +2,8 @@
 
 # 0. Prepare images
 # only debian 12 (bookworm) has latest ffmpeg
-ARG DEBIAN_VERSION="bookworm-slim"
-ARG GO_VERSION="1.20-buster"
+ARG DEBIAN_VERSION="bullseye-slim"
```

Comment:
why go back to bullseye if only bookworm has the latest ffmpeg?

Reply:
make sense

Reply:
thanks for advancing some of my favourite projects